.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ templates:
       - UROOT_SOURCE: /go/src/github.com/u-root/u-root
       - CGO_ENABLED: 0
       # x7 all timeouts for QEMU VM tests since they run without KVM.
-      - UROOT_QEMU_TIMEOUT_X: 7
+      - UROOT_QEMU_TIMEOUT_X: 10
```

Comment:
Is this still needed now that we disabled arm? This number means that the timeout for tests is 10x slower on circleci compared to local.

Reply:
we did not disable arm. But if we want I can set it back.

Reply:
I set it back. Let's see if it still passes :-)